src/Paysera/WalletApi/Mapper.php

```diff
@@ -2387,4 +2403,51 @@ public function encodeTransferConfiguration(Paysera_WalletApi_Entity_TransferCon
     {
         return (new Paysera_WalletApi_Mapper_TransferConfigurationMapper())->mapFromEntity($transferConfiguration);
     }
+
+    /**
+     * Decodes client wallet permissions
+     *
+     * @param array $data
+     *
+     * @return Paysera_WalletApi_Entity_ClientWalletPermissions
+     */
+    public function decodeClientWalletPermissions(array $data)
+    {
+        $scopes = [];
+
+        $walletPermissions = Paysera_WalletApi_Entity_ClientWalletPermissions::create()
+            ->setWalletId($data['id'])
+            ->setAccountNumber($data['account'])
+        ;
+
+        foreach ($data as $key => $value) {
+            switch ($key) {
+                case Paysera_WalletApi_Entity_ClientWalletPermissions::SCOPE_BALANCE:
```

Comment:
why there is such case? can't it be other permissions?

In case we say, that it's just for balance and statements, it must be clear. In case it's some "security" measure, then it's wrong, as everyone can change such "case"

Reply:
Done